main.js

```diff
@@ -243,28 +243,27 @@ const connectAndEndorse = async (browser, link, iter) => {
     console.log('past navigation');
 
     // now endorse!
-    await page.evaluate(() => {
+    await page.evaluate(async () => {
       // credit to Julia, Mike, and Augusto for the below code. You sparked the idea to create this repo!
-      setTimeout(async () => {
-        // Once on the skills page, scroll to the bottom:
-        let prev = 0;
-        let cur = document.body.scrollHeight;
-        while (cur !== prev) {
-          window.scrollTo(0, document.body.scrollHeight);
-          await asyncWait(500);
```

Comment:
This call is to a function outside the scope of the DOM which is likely throwing an error.